lib/enumerize/base.rb

```diff
@@ -6,6 +6,14 @@ def self.included(base)
       if base.respond_to?(:validate)
         base.validate :_validate_enumerized_attributes
       end
+
+      class << base
+        if method_defined?(:inherited) && (not method_defined? :inherited_without_enumerized)
+          alias_method :inherited_without_enumerized, :inherited
+        end
+
+        alias_method :inherited, :inherited_with_enumerized
```

Comment:
I think that it might be necessary to move this `alias_method` inside the `if` condition above.